js_modules/dagit/packages/core/src/workspace/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -89,9 +77,22 @@ export const AssetGraphExplorer: React.FC<Props> = (props) => {
     const queryAssetNodes = assetNodes.filter((a) =>
       queryOps.all.some((op) => op.name === a.opName),
     );
+    const qraphData = buildGraphData(queryAssetNodes, explorerPath.pipelineName);
```

Comment:
kind of confusing to have `graphData` and `qraphData`

Reply:
Oh god will fix... I actually spent 30 min staring at this change because I was exporting qraphdata from this function and TS didn't catch the spelling issue. Will fix it here too